tests/MongoDB.Driver.Core.Tests/Core/Servers/ServerTests.cs

```diff
@@ -546,15 +557,17 @@ internal void ShouldInvalidateServer_should_return_expected_result_for_exception
             var serverId = new ServerId(clusterId, new DnsEndPoint("localhost", 27017));
             var connectionId = new ConnectionId(serverId);
             var command = new BsonDocument("command", 1);
-            var commandResult = new BsonDocument("ok", 1);
+            var notMasterResult = new BsonDocument {{ "code", ServerErrorCode.NotMaster}};
+            var nodeIsRecoveringResult = new BsonDocument("code", ServerErrorCode.InterruptedAtShutdown);
```

Comment:
We need a more detailed commandResult so that `ShouldInvalidateServer` logic can be properly tested.